src/pages/BtecGuide/index.tsx

```diff
@@ -0,0 +1,610 @@
+                      inline
+                      to="https://eips.ethereum.org/EIPS/eip-2334#eth2-specific-parameters"
+                    >
+                      <FormattedMessage defaultMessage="EIP-2334" />
+                    </Link>
+                  ),
+                  docs: (
+                    <Link
+                      primary
+                      inline
+                      to="https://github.com/ethereum/staking-deposit-cli#step-2-create-keys-and-deposit_data-json-2"
+                    >
+                      <FormattedMessage defaultMessage="docs" />
+                    </Link>
+                  ),
+                }}
+              />
+            </Text>
+          </Alert>
+          <section>
+            <Anchor to="#execute-with-params" id="execute-with-params">
+              <SubSectionTitle level={3} className="mb10">
+                <FormattedMessage defaultMessage="Execute with params" />
+              </SubSectionTitle>
+            </Anchor>
+            <Alert variant="success">
+              <Text>
+                <FormattedMessage
+                  defaultMessage="You can find {docsHere}."
+                  values={{
+                    docsHere: (
+                      <Link
+                        primary
+                        inline
+                        to="https://github.com/ethereum/staking-deposit-cli/blob/bls-to-execution-change/README.md#generate-bls-to-execution-change-arguments"
+                      >
+                        <FormattedMessage defaultMessage="docs of full arguments here" />
+                      </Link>
+                    ),
+                  }}
+                />
+              </Text>
+            </Alert>
+            <section>
+              <Anchor
+                to="#execute-with-params-option-1"
+                id="execute-with-params-option-1"
+              >
+                <SubSectionTitle level={4} className="mb10">
+                  <FormattedMessage defaultMessage="Option 1: binary files" />
+                </SubSectionTitle>
+              </Anchor>
+              <Text className="mb10">
+                <strong>
+                  <FormattedMessage defaultMessage="Interactive mode" />
+                </strong>
+              </Text>
+              <CodeBlock>./deposit generate-bls-to-execution-change</CodeBlock>
+              <Text className="mb10">
+                <strong>
+                  <FormattedMessage defaultMessage="Command line with flags" />
+                </strong>
+              </Text>
+              <CodeBlock className="indent">
+                <span>
+                  ./deposit <span className="dimmed">--language=english</span>{' '}
+                  generate-bls-to-execution-change \
+                </span>
+                <br />
+                <span className="dimmed">{`--chain=mainnet \\`}</span>
+                <br />
+                <span className="dimmed">
+                  --mnemonic="
```

Comment:
@hwwhww Curious your thoughts on removing this line to default users to enter their mnemonic as a prompt only, preventing it from ever ending up in the terminal history. (Similar in the other location this is used below)